orangecontrib/infrared/widgets/owpreproc.py

```diff
@@ -704,8 +706,12 @@ def __call__(self, data):
             elif self.method == self.Offset:
                 data.X -= np.min(y_s, axis=1, keepdims=True)
             elif self.method == self.Attribute:
-                # Not implemented
-                pass
+                # attr normalization applies to entire spectrum, regardless of limits
+                # meta indices are -ve and start at -1
+                if self.attr not in (None, "None", ""):
```

Comment:
None is the default attribute, "None" is the string the widget sets as default, and "" is what the widget VariableListModel returns when there are no meta attributes. I agree this is messy but I liked having something in the combobox when there is no data/attribute to select. I'm not tied to it though.